test/defer.js

```diff
@@ -0,0 +1,146 @@
+const { test } = require('tap')
+const Fastify = require('fastify')
+const mercurius = require('../index')
+const { canUseIncrementalExecution } = require('../lib/util')
+
+if (canUseIncrementalExecution) {
+  const schema = `
+    directive @defer(
+      if: Boolean! = true
+      label: String
+    ) on FRAGMENT_SPREAD | INLINE_FRAGMENT
```

Comment:
I would do this behind a `defer: true` option, set to false by default.

Reply:
Hey @mcollina I've added `opts.defer`, but I'm not quite sure if we really need that, since we decided to cut this off to a new release branch where we assume `@defer` will be available to everyone.

I can remove `opts.defer` and just leave its directive definition so that we always have this functionality with `graphql@17`.

What do you think?